src/main/java/com/commercetools/sync/cartdiscounts/utils/CartDiscountReferenceResolutionUtils.java

```diff
@@ -42,26 +42,28 @@
    *   </tbody>
    * </table>
    *
-   * <p><b>Note:</b> The {@link Type} reference should has an expanded reference with a key. Any
-   * reference that is not expanded will have its id in place and not replaced by the key will be
-   * considered as existing resources on the target commercetools project and the library will
-   * issues an update/create API request without reference resolution.
+   * <p><b>Note:</b> The {@link Type} reference should contain Id in the map(cache) with a key
+   * value. Any reference that is not available in the map will have its id in place and not
+   * replaced by the key will be considered as existing resources on the target commercetools
```

Comment:
```suggestion
   * value. Any reference, which have its id in place and not replaced by the key, it would not be found in the map. In this case, this reference will be considered as existing resources on the target commercetools
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/701/commits/f5c3bab5c60d90c604ebdc2fbb84cbf59eb486b8

Reply:
i think the sentence should be like this : 

> Any reference, which have its id in place and not replaced by the key, it would not be found in the map.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/701/commits/9d05a0c46c2942e2f2af5d445633b964d67155ed